libvmi/driver/xen/xen_events_private.h

```diff
@@ -144,6 +144,15 @@ status_t xen_set_cpuid_event(vmi_instance_t vmi, bool enabled);
 status_t xen_start_single_step(vmi_instance_t vmi, single_step_event_t *event);
 status_t xen_stop_single_step(vmi_instance_t vmi, uint32_t vcpu);
 status_t xen_shutdown_single_step(vmi_instance_t vmi);
+status_t xen_altp2m_get_domain_state (vmi_instance_t vmi, bool *state);
```

Comment:
Does the new header require the copyright/licences like the xen_events_private.h
[...]

> * Portions of this header and dependent code is based upon that in xen-access,
> *    from the official Xen source distribution.  That code carries the
> *    following copyright notices and license.
> *
> * Copyright (c) 2011 Virtuata, Inc.
> * Copyright (c) 2009 by Citrix Systems, Inc. (Patrick Colp), based on
> *   xenpaging.c
[...]